types/setting.go

```diff
@@ -252,11 +252,12 @@ var (
 
 	SettingDefinitionGuaranteedEngineCPU = SettingDefinition{
 		DisplayName: "Guaranteed Engine CPU",
-		Description: "(EXPERIMENTAL FEATURE) Allow Longhorn Engine to have guaranteed CPU allocation. The value is " +
+		Description: "(EXPERIMENTAL) Allow Longhorn Engine to have guaranteed CPU allocation. The value is " +
 			"how many CPUs should be reserved for each Engine/Replica Manager Pod created by Longhorn. For example, " +
 			"0.1 means one-tenth of a CPU. This will help maintain engine stability during high node workload. It " +
 			"only applies to the Engine/Replica Manager Pods created after the setting took effect. WARNING: " +
-			"Attaching of the volume may fail or stuck while using this feature due to the resource constraint. " +
+			"Attaching the volume may fail or become stuck while using this feature due to the resource constraint. " +
```

Comment:
I forgot to remove this line. This option no longer results in volume failure to attach, but the user needs to manually restart the instance managers still.